src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -60,9 +61,12 @@ public HelpWindow() {
         helpBox.initModality(Modality.NONE);
         helpBox.setResizable(true);
         helpBox.getDialogPane().setPrefWidth(900.0);
-        helpBox.getDialogPane().setPrefHeight(530.0);
+        helpBox.getDialogPane().setPrefHeight(450.0);
         helpBox.getDialogPane().getStylesheets().add("/view/DarkTheme.css");
         helpBox.getDialogPane().getStyleClass().add("helpBox");
+        helpBox.getDialogPane().setStyle("-fx-font-family: 'Courier New';");
+        helpBox.setX(200);
+        helpBox.setY(125);
```

Comment:
Works for me. Is that on master? I see in your PR you put back `Courier New`
![untitled1000](https://cloud.githubusercontent.com/assets/109479/20049808/6c67ffd0-a500-11e6-87fa-09b277d05339.png)